AutocompleteClient/FW/Logic/Result/Responses/CIOQuizResultsResponse.swift

```diff
@@ -13,12 +13,56 @@ import Foundation
  */
 public struct CIOQuizResultsResponse {
     /**
-     Quiz result
+     List of facets returned for the result set
      */
-    public let result: CIOQuizResult
+    public let facets: [CIOFilterFacet]
+
+    /**
+     List of groups (categories) returned for the result set
+     */
+    public let groups: [CIOFilterGroup]
+
+    /**
+     List of results returned for the browse query
+     */
+    public let results: [CIOResult]
+
+    /**
+     List of sorting options
+     */
+    public let sortOptions: [CIOSortOption]
+    
+    /**
+     Total number of results for the result
+     */
+    public let totalNumResults: Int
     
     /**
-     Version Id of the quiz result 
+     Sources of the result set
+     */
+    public let resultSources: CIOResultSources?
+    
+    /**
+     Result ID of the result set returned
+     */
+    public let resultID: String
+    
+    /**
+     Specific quiz_version_id for the quiz. 
+     Version ID will be returned with the first request and it should be passed with subsequent requests.
+     More information can be found: https://docs.constructor.io/rest_api/quiz/using_quizzes/#quiz-versioning
```

Comment:
How do you feel about rephrasing this a little:

`Unique version id for the quiz. The quiz version id will be returned with the first...`

If that sounds good, let's make adjustments to the other places where we have a comment for `quizVersionId` and `quizSessionId` 👍